components/eam/tools/topo_tool/cube_to_target/cube_to_target.F90

```diff
@@ -856,9 +825,6 @@ program convterr
     
     wt = weights_all(count,1)
     
-    if (lzero_out_ocean_point_phis.AND.landfrac_target(i).lt.0.01_r8) then
```

Comment:
This is kind of interesting, I hadn't noticed this before. This logical appears to set the new elevation to zero if landfrac is below a threshold value. Maybe this was the purpose for the landfrac variable in the original version. Do you see a problem with removing this? The hard-coded threshold seems kind of problematic because that would probably be resolution dependent.

Reply:
yes this flag is one of the reasons why the NCAR repo keeps the landfrac (https://github.com/NCAR/Topo/issues/19) 

i had np problem after removing ``lzero_out_ocean_point_phis'' because i've never enabled this flag before